src/stories/Library/event-list-item/EventListItem.tsx

```diff
@@ -0,0 +1,49 @@
+// EventListItem.tsx
+import { ReactComponent as ArrowSmallRight } from "../Arrows/icon-arrow-ui/icon-arrow-ui-small-right.svg";
+import { Tag } from "../tag/Tag";
+
+export type EventListItemProps = {
+  image: string;
+  title: string;
+  description: string;
+  date: string;
+  time: string;
+  location: string;
+  price: string;
+  href: string;
+};
+
+export const EventListItem: React.FC<EventListItemProps> = ({
+  image,
+  title,
+  description,
+  date,
+  time,
+  location,
+  price,
+  href,
+}) => {
+  return (
+    <a href={href} className="event-list-item arrow__hover--right-small">
+      <div className="event-list-item__image-container">
+        <img src={image} alt={title} className="event-list-item__image" />
+      </div>
+      <div className="event-list-item__content">
+        <Tag hasBackground className="event-list-item__tag">
+          foredrag
+        </Tag>
+        <p className="event-list-item__date">{date}</p>
+        <h2 className="event-list-item__title">{title}</h2>
+        <p className="event-list-item__description">{description}</p>
+        <div className="event-list-item__location-wrapper">
+          <p className="event-list-item__location">{location}</p>
+        </div>
+        <div className="event-list-item__schedule">
+          <time className="event-list-item__time">{time}</time>
```

Comment:
Consider whether it is appropriate to add a [`datetime` attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/time) for the `<time>` element - and whether this or the date is the right element to do it.

I know that adding the correct value corresponding to the displayed will be difficult but with a comment it might be OK to hard code the value to show that it is important to actually have it set.